redirects.csv

```diff
@@ -2,7 +2,9 @@
 /group-renew groupIds=:groupIds,/group-renew?groupIds=:groupIds,200! Role=user
 /group-renew groupIds=:groupIds,/signin?redirectUrl=/group-renew?groupIds=:groupIds,302!
 http://crossroads.net/*,https://www.crossroads.net/:splat,301!,master
+https://crossroads.net/*,https://www.crossroads.net/:splat,301!,master
 http://${env:CRDS_APP_DOMAIN}/*,https://${env:CRDS_APP_DOMAIN}/:splat,301!
+https://${env:CRDS_APP_DOMAIN}/*,https://${env:CRDS_APP_DOMAIN}/:splat,301!
```

Comment:
Not sure what you're trying to accomplish here but this rule will create an infinite loop. 

`CRDS_APP_DOMAIN == 'www.crossroads.net'`

Reply:
@tcmacdonald I did this to cover both http and https